js/package.json

```diff
@@ -96,10 +96,9 @@
     "ts-jest": "27.0.0",
     "ts-node": "10.0.0",
     "typedoc": "0.20.36",
-    "typescript": "4.0.2",
+    "typescript": "4.3.3",
```

Comment:
🎉

Reply:
Will need to figure out a solution to this before proceeding: https://github.com/microsoft/TypeScript-DOM-lib-generator/pull/890#issuecomment-862866183

Reply:
We could try https://devblogs.microsoft.com/typescript/announcing-typescript-4-5/#esm-nodejs